docs/project-backlog.md

```diff
@@ -0,0 +1,122 @@
+* `available` for actionable work items.
+* `needs-discussion` for issues which cannot be triaged quickly, but raise important points.
+* `no-further-action` for issues which can be quickly resolved.
+
+### Available
+label: `available`
+
+For new work requests Once it has been classified, a ticket is up for grabs to be picked up to be worked on by a contributor.
+
+All work is labeled with a priority:
+* `P0` = Critical (Drop everything and work on it.)
+* `P1` = Important (Work on it if no P0 are present. Use judgment to prioritize items within a list of P1s)
+* `P2` = Low (Work on it when no P1 are left or some one from community wants to contribute)
+
+Work can also be classified into a number of categories, to make available issues easier to find. Common examples of work categories are:
+* `feature-request`
+* `enhancement`
+* `bug`
+* `docs`
+
+Available work can be assigned in the following ways:
+* A contributor assigns the ticket to themselves.
+* A maintainer assigns the ticket to a contributor.
+
+All tickets transition to `wip` once they have been assigned.
+
+### WIP
+label: `wip`
+
+Any ticket which has been assigned a lead contributor is considered work in progress. 
+
+* If a PR is merged which resolves the issue, the ticket can be closed as `completed`.
+* If work is blocked on review, the ticket transitions to `needs-review`.
+* If work is abandoned, or the ocntributors otherwise change their minds, the ticket can be closed as `no-further-action`.
+
+### Needs Review
+label: `needs-review`
+
+Implementation work related to the ticket has been completed. Work related to a Review and/or any changes requested as part of the review  is in progress.  Issues with a PR associated with it will be considered as in ‘In Review’.
+
+* Move the issue back to `in-progess` if significant changes are needed.
+* Close the issue with the `completed` label.
+* Close the issue with the `no-action` label.
+
+### Needs Discussion
+label: `needs-discussion`
+
+Some issues are not directly related to a particular code change. This can often arise with tickets describing unclear edge cases, and situations where applying OpeTelemetry is proving to be difficult.  If a ticket is worth considering in the issue backlog, but not scoped clearly enough to move to `wip`, then it `needs-discussion`.
+
+* When possible, move the discussion forwards by using code examples.
+* Mark the ticket as `agreed` if a resolution is found. 
+* Close the ticket as `no-action` if discussion ends without a solution.
+* Note: If the ticket is a help request with an existing answer, please link to the answer before closing the ticket.
+
+### Agreed
+label: `agreed`
+
+An agreement has been reached. Once a course of action is determined, discussion is (hopefully) over.
+
+* Clearly list any resulting action items which need to be adressed.
+* Close the ticket once all action items are completed.
+
+### Stale
+label: `stale`
+
+When a ticket has violated an SLA, it becomes stale.
```

Comment:
do you envision some automation in place?

Reply:
For SLAs, yes. But it should be possible to do by hand, or it is too complicated. 

For next step, I suggest we try setting up project boards for java and c#, and model some of this workflow.

Reply:
this SLA tag is clearly have some target in mind. You want to ensure "healthy" project with timely replies. If somebody will do it by hand - repo is already "healthy". Your issue will be read and classified.

So from my perspective SLA like this can mostly be used as a metric for transparency or reminder to the maintainer with suggested actions. This is how I read the proposal in your document.

Comment with the mention of maintainer and triager like this can be posted on the issue:

Like *this isssue was untriaged for a long time. (at)poster - please make sure the ticket has enough details and, when possible, clear steps to repro. (at)repo-maintainers If you feel that some details are missing - add this "label". If it takes too much time to reproduce - please indicate that you are looking into it using this "label"., etc...*

But this thing will only be useful with automation